exercises/concept/lucas-numbers/.docs/hints.md

```diff
@@ -0,0 +1,26 @@
+# Hints
+
+## General
+
+- Use the built-in [(linked) list type][list].
+- Use the built-in [Stream][stream] module functions to create a stream
+
+## 1. Generate the base cases
+
+- You can use multiple [function clauses][multiple-fn-clauses] and [pattern-matching][pattern-matching] to create the base case functions.
+
+## 2. Create the generalized case
+
+- Use a [Stream][stream] function to generate a sequence of numbers, with the next being created from the previous.
+- The starting numbers are `2` then `1`, which you can pass in one parameter using a tuple to make a pair `{2, 1}`
```

Comment:
I think this hint is very specialized and only makes sense for a few Stream functions. We're not naming any specific functions in the hints for this step, which I think might be a problem for some students. Maybe we should point to https://hexdocs.pm/elixir/Stream.html#iterate/2 specifically? The first hint sounds like it could link to `iterate` directly instead of the whole module.

Reply:
Sounds good, is this better? https://github.com/exercism/elixir/pull/688/commits/115a579698bf2b7b529b3de76de3c729eec2cb87